backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -50,4 +54,21 @@ public UserModel getUserById(
 		return userModelAssembler.toModel(user);
 	}
 
+
+    @PostMapping({ "/verifyCode/"})
+    @Operation(summary = "Verify the status of a confirmation code.", operationId="verify-confirmation-code")
+    public ConfirmationCodeStatus getConfirmationCodeStatus(
+            @NotBlank(message = "code must not be null or blank")
+            @Parameter(description = "The confirmation code.", required = true)
+            @RequestParam ConfirmationCode code,
```

Comment:
Should `code` be a `String` rather than the entire `ConfirmationCode` entity? From the POV of a possible API consumer like a front-end web application, I wouldn't know what to pass for the other fields (like `expiryDate`, `lastModifiedBy`) of the `ConfirmationCode` aside from the code itself.

Reply:
This is still unresolved.

Reply:
Implemented in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1.

Reply:
Although `@RequestParam` has been changed to `@RequestBody`, it does not match the proposed format of
``` json
{
  "confirmationCode": "aabbccdd"
}
```

Consider creating a new model class for email verifications that is annotated with `@RequestBody` instead.

Reply:
Correct. Request and response bodies should be valid JSON.
<https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples#1usejsonastheformatforsendingandreceivingdata>

Reply:
As of commit 70d88ff21299e8b6f77785dd0b22d60b10f1a733 the `EmailVerificationModel` has been created and used in this context.